.eslintignore

```diff
@@ -0,0 +1,6 @@
+node_modules/*
+dist/*
+
+examples/
+  dist/*
+  node_modules/*
```

Comment:
This block isn't necessary. Also, I think ESLint parses this file like a `.gitignore` file rather than a YAML file. I believe using `**/dist/*` will cover all `dist` directories in the repo.